diff -ru util-linux-2.19.1/fdisk/Makefile.am util-linux-2.19.1-fixtinfo/fdisk/Makefile.am
--- util-linux-2.19.1/fdisk/Makefile.am	2011-05-02 11:10:08.000000000 +0200
+++ util-linux-2.19.1-fixtinfo/fdisk/Makefile.am	2011-12-13 09:53:59.626285046 +0100
@@ -65,7 +65,7 @@
 dist_man_MANS += cfdisk.8
 cfdisk_SOURCES = cfdisk.c $(fdisk_common)
 cfdisk_CFLAGS = $(cflags_blkid)
-cfdisk_LDADD = @NCURSES_LIBS@ $(ldadd_blkid)
+cfdisk_LDADD = @NCURSES_LIBS@ -ltinfo $(ldadd_blkid)
 endif
 endif
 
Only in util-linux-2.19.1-fixtinfo/fdisk: Makefile.am~
